src/SFML/Audio/SoundFileReaderWav.cpp

```diff
@@ -272,12 +286,54 @@ bool SoundFileReaderWav::parseHeader(Info& info)
             }
             m_bytesPerSample = bitsPerSample / 8;
 
-            // Skip potential extra information (should not exist for PCM)
-            if (subChunkSize > 16)
+            if (format == WaveFormatExtensible)
             {
-                if (m_stream->seek(m_stream->tell() + subChunkSize - 16) == -1)
+                // Extension size
+                Uint16 extensionSize = 0;
+                if (!decode(*m_stream, extensionSize))
+                    return false;
+
+                // Valid bits per sample
+                Uint16 validBitsPerSample = 0;
+                if (!decode(*m_stream, validBitsPerSample))
+                    return false;
+
+                // Channel mask
+                Uint32 channelMask = 0;
+                if (!decode(*m_stream, channelMask))
+                    return false;
+
+                // Subformat
+                char subformat[16];
+                if (m_stream->read(subformat, sizeof(subformat)) !=
+                        sizeof(subformat))
```

Comment:
Could be on one line.

Reply:
Do you guys have an established max line length?

Reply:
Readability over strict max line length IMO.

Reply:
Mario is right. But we actually don't have such rule in our [style guide](https://www.sfml-dev.org/style.php).